lib/bambora/adapters/multipart_mixed_request.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Bambora
+  module Adapters
+    ##
+    # Creates headers and a body for a multipart/mixed request with a file and a JSON body.
+    class MultipartMixedRequest
+      attr_reader :multiparty
+
+      def initialize(options = {})
+        @multiparty = Multiparty.new { |party| party.parts = options[:multipart_args] }
```

Comment:
Seems like there is a lot of heavy lifting going on in this `Multiparty` functionality 💪.